realsence_track_person.py

```diff
@@ -31,6 +31,7 @@
 # ストリーミング開始
 pipeline = rs.pipeline()
 profile = pipeline.start(config)
+align = rs.align(rs.stream.color)
```

Comment:
After we use the align process, the pixel of the nose in the color image is exactly the nose's pixel in the depth image. So we don't need to calculate the center_mask to get an average depth of one point, just directly output the depth of that point.

Reply:
Oh, that's it!  How nice!

Reply:
This align process is nice right?